charts/istio-base-1.19.9/Chart.yaml

```diff
@@ -12,3 +12,4 @@ sources:
   - https://github.com/istio/istio
 engine: gotpl
 icon: icons/android-192x192.png
+kubeVersion: '< 1.31.0'
```

Comment:
Supported Kubernetes versions https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases
1.20 -> 1.25, 1.26, 1.27, 1.28, 1.29
1.19 -> 1.25, 1.26, 1.27, 1.28

Reply:
I would love to live in a world where every component in Oracle Cloud Native Environment had a well-aligned support matrix.  That's not he world we live in.  We need to support up to 1.30 to account for upgrades from 1.8 and 1.9 with some slop to give people flexibility.